Jobs/Engine/Push/Pushalot.cs

```diff
@@ -0,0 +1,96 @@
+﻿// This file is part of AlarmWorkflow.
+// 
+// AlarmWorkflow is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+// 
+// AlarmWorkflow is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+// 
+// You should have received a copy of the GNU General Public License
+// along with AlarmWorkflow.  If not, see <http://www.gnu.org/licenses/>.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Mime;
+using System.Text;
+using System.Web;
+using AlarmWorkflow.Shared.Diagnostics;
+
+namespace AlarmWorkflow.Job.PushJob
+{
+    public static class Pushalot
+    {
```

Comment:
Bitte **public** Member immer mit XML-Dokumentaren kommentieren. In diesem Fall hier denke ich, dass dies eh nicht notwendig ist. Bitte häng noch einen Commit dran, um das auf "static class Pushalot" zu ändern, damit die Konventionen eingehalten werden.

In Zukunft dann bitte immer gleich auf die Coding Guidelines achten.